MovieApp/PresentationLayer/Favorites/FavoritesViewController.swift

```diff
@@ -11,3 +16,68 @@ class FavoritesViewController: UIViewController {
     }
 
 }
+
+extension FavoritesViewController: UICollectionViewDataSource {
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        numberOfItemsInSection section: Int
+    ) -> Int {
+        100
+    }
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        cellForItemAt indexPath: IndexPath
+    ) -> UICollectionViewCell {
+        guard
+            let cell = movieCollectionView.dequeueReusableCell(
+                withReuseIdentifier: MoviePosterCell.cellIdentifier,
+                for: indexPath) as? MoviePosterCell
+        else {
+            let cell = MoviePosterCell()
+            return cell
+        }
+
+        let movie = MovieViewModel(
```

Comment:
I know this is just mocked data but we never mock the data in the VC. If there is no business logic for this (UC layer or below), we mock the data in the presenter so that the VC is already set up when we connect the presenter with the UC.